pkg/reconciler/autoscaling/kpa/scaler_test.go

```diff
@@ -148,12 +149,37 @@ func TestScaler(t *testing.T) {
 		label:         "waits to scale to zero after idle period (custom PA window)",
 		startReplicas: 1,
 		scaleTo:       0,
-		wantReplicas:  0,
-		wantScaling:   false,
 		paMutation: func(k *pav1alpha1.PodAutoscaler) {
 			WithWindowAnnotation(paStableWindow.String())(k)
 			paMarkActive(k, time.Now().Add(-paStableWindow))
 		},
+		wantReplicas: 0,
+		wantScaling:  false,
+	}, {
+		label:         "scale to zero after grace period, but before last pod retention",
```

Comment:
I found it hard to get the expected behavior from the test label. Actually this test case should not scale to zero. How about `can't scale to zero after grace period, but before last pod retention`

Reply:
Done